### ISSUE TITLE
Problem with MultitPEN launching

### ISSUE THREAD

**Issue by dvpolyakov** [EarlhamInst/MultiPEN#2]
When I try launch example code from user manual ("...handout.pdf") in the command line like this:

$mcrpath="mcrroot/"
$OutputDirectory="ExampleOutputs/"
$ExpressionData="ExampleInputs/expressionData.txt"
$Interactions="ExampleInputs/interactionMatrix.txt"
$SampleClass="ExampleInputs/sampleClass.txt"
$lambda=0.001

and then

$./MultiPEN $mcrpath FeatureSelection $OutputDirectory $ExpressionData $Interactions $SampleClass $lambda

I have an error:

./MultiPEN: error while loading shared libraries: libmwlaunchermain.so: cannot open shared object file: No such file or directory

I think there is a problem with path to "MATLAB Runtime" folder

**Comment by wjurkowski** [comment]
yes, you need to set up a path to where your Matlab runtime is installed. The script to do it is run_MultiPEN.sh
